### PR TITLE
Display product

### PR DESCRIPTION
**Pull request by annamarika**
Created first product and added styles components, storybook and testing.

### CODE REVIEW

File: pages/index.js [hunk 1]
```diff
@@ -1,66 +1,30 @@
...
+		<PageContainer>
+			<Product products={products} />
+		</PageContainer>
 	);
 }
+
+const PageContainer = styled.div`
+	margin: 22px;
+	display: flex;
+	flex-direction: row;
```
Comment (mbosselmann) [inline pages/index.js:26]: `flex-direction: row` is the default for `flex-direction`. Therefore, you don't need to write this declaration. :)

File: src/components/Button/Button.stories.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+			return (
+				<div style={{ padding: '3em', maxWidth: '400px', position: 'relative' }}>
+					<Story />
+				</div>
+			);
+		},
+	],
+};
+
+const props = { props: { email: 'hallo@hallo.de' } };
```
Comment (mbosselmann) [inline src/components/Button/Button.stories.js:19]: If I did understand correclty, then is `EmailButton` a styled component where you don not work with anything `props` related yet. Do you really need `const props` for your `Story`? 🤔

File: src/components/GlobalStyle/GlobalStyle.js [hunk 3]
```diff
@@ -0,0 +1,25 @@
+import { createGlobalStyle } from 'styled-components';
+
+const GlobalStyle = createGlobalStyle`
+html {
+    font-size: 62.5%;
+	font-family: -apple-system, BlinkMacSystemFont, Segoe UI, Roboto, Oxygen,
+    Ubuntu, Cantarell, Fira Sans, Droid Sans, Helvetica Neue, sans-serif;
+  }
```
Comment (mbosselmann) [inline src/components/GlobalStyle/GlobalStyle.js:8]: The code formatting didn't work here.

File: src/components/GlobalStyle/GlobalStyle.js [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+    margin: 0px;
+    padding: 0px;
+  }
+  
+  body {
+	  
+    background-color: #E2CADF;
+    
+    font-size: 1.5rem;
+  }
```
Comment (mbosselmann) [inline src/components/GlobalStyle/GlobalStyle.js:21]: Just nit-picking: You should remove empty lines that aren't necessary. :)

File: src/components/Product/StyledProduct.js [hunk 5]
```diff
@@ -0,0 +1,36 @@
+import styled from 'styled-components';
+
+const ProductContainer = styled.div`
```
Comment (mbosselmann) [inline src/components/Product/StyledProduct.js:3]: Your `Product` component renders all your products or? To have more semantic code you could work with a `ul` element where you map your `products` in a `Product` component that is a `li` element.

File: src/components/Product/StyledProduct.js [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+	width: 250%;
+	height: 386px;
+	position: relative;
+	object-fit: cover;
+	border-radius: 50px;
+	overflow: hidden;
+	border: 3px solid #4b587f;
+`;
+
+const ProductArticle = styled.div`
```
Comment (mbosselmann) [inline src/components/Product/StyledProduct.js:22]: And I have a question: In my opinion it is not necessary to have `Product` as part of the name of every styled-component for your `Product` component. Did you decide this in your course that it is useful? If not I would recommend to only work with `Article`, `Container` etc. as names.

**Review by mbosselmann — COMMENTED** [review]
Good job! 👍 I left some remarks you should consider before merging. :)

File: src/components/Product/StyledProduct.js (hunk 6 above)
Comment (annamarika) [reply src/components/Product/StyledProduct.js:22]: You are right, I don't need the Product as part of the styled-component. Sometimes we used it, but only when there were more than one Article, Container etc. I changed it

File: src/components/Product/Product.spec.js [hunk 7]
```diff
@@ -0,0 +1,28 @@
+import Product from './Product';
+import { render, screen } from '@testing-library/react';
+import '@testing-library/jest-dom';
+
+describe('ProductPage', () => {
+	it('should render one Button, a text and an image', () => {
```
Comment (PReichetanz) [inline src/components/Product/Product.spec.js:6]:
```suggestion
	it('renders one button, a text and an image', () => {
```

File: src/components/Product/Product.js [hunk 8]
```diff
@@ -0,0 +1,26 @@
...
+				<Image
+					src="/test.jpg"
+					alt="dress"
+					width={3456}
+					height={5184}
+					layout="fill"
+					objectFit="cover"
+				/>
+			</ImageWrapper>
+			<Article>
```
Comment (PReichetanz) [inline src/components/Product/Product.js:18]: I am confused that the "Article" styled component is a div and not an article element.

File: src/components/Product/StyledProduct.js [hunk 9]
```diff
@@ -0,0 +1,36 @@
...
+	width: 250%;
+	height: 386px;
+	position: relative;
+	object-fit: cover;
+	border-radius: 50px;
+	overflow: hidden;
+	border: 3px solid #4b587f;
+`;
+
+const Article = styled.div`
```
Comment (PReichetanz) [inline src/components/Product/StyledProduct.js:22]: That's not the best naming choice, I would say.

**Review by PReichetanz — COMMENTED** [review]
Nice work, I only have some minor issues to consider 🚀

**Review by mbosselmann — APPROVED** [review]
👍

**Pull request merged**